.github/CODEOWNERS

```diff
@@ -444,7 +444,6 @@ lerna.json @grafana/frontend-ops
 /public/robots.txt @grafana/frontend-ops
 /public/sass/ @grafana/grafana-frontend-platform
 /public/test/ @grafana/grafana-frontend-platform
-/public/testdata/ @grafana/grafana-frontend-platform
```

Comment:
Is this deleted? 🤔

Reply:
Yes the folder doesn't exist anymore:
https://github.com/grafana/grafana/tree/f6b9f37839a153403c209cde6a44bdef6eb73121/public

Reply:
correct, this folder only contained the CSV files for the test data source.